src/components/mgt-person/mgt-person.ts

```diff
@@ -301,13 +301,13 @@ export class MgtPerson extends MgtTemplatedComponent {
     }
 
     const isLarge = this.showEmail && this.showName;
-    const smallAvatar = this.smallAvatar;
+    const isSmallAvatar = this.avatarSize === 'small' ? true : false;
```

Comment:
You can simplify this:

```suggestion
    const isSmallAvatar = this.avatarSize === 'small';
```